src/mpi/ccl/Makefile.mk

```diff
@@ -0,0 +1,7 @@
+##
+## Copyright (C) by Argonne National Laboratory
+##     See COPYRIGHT in top-level directory
+##
+
+mpi_core_sources += \
+    src/mpi/ccl/cclcomm.c
```

Comment:
Place it under `src/util` or `src/mpid/common`

Reply:
What is the logic we should use to decide where to store these new files? Note there are now a few new source files in this directory.

Reply:
Got it, thanks for the explanation! I moved the `ccl` directory to `src/util`.